cmd/crc/cmd/delete.go

```diff
@@ -52,7 +54,8 @@ func deleteMachine(client machine.Client, clearCache bool, cacheDir string, inte
 		return false, errors.New("non-interactive deletion requires --force")
 	}
 
-	yes := input.PromptUserForYesOrNo("Do you want to delete the OpenShift cluster", force)
+	yes := input.PromptUserForYesOrNo(fmt.Sprintf("Do you want to delete the %s", crcPreset.UserFriendlyName(crcConfig.GetPreset(config))),
+		force)
```

Comment:
For what it's worth, building strings this way must not be done when you want to translate your application, "the" might require a different translation depending on what '%s' is replaced with . We are not translating crc, so this is mostly theoritical :)